src/librustc_middle/mir/interpret/error.rs

```diff
@@ -384,7 +393,7 @@ pub enum UndefinedBehaviorInfo<'tcx> {
     /// Using a string that is not valid UTF-8,
     InvalidStr(std::str::Utf8Error),
     /// Using uninitialized data where it is not allowed.
-    InvalidUninitBytes(Option<Pointer>),
+    InvalidUninitBytes(Option<UninitBytesAccess>),
```

Comment:
This looks like it makes the variant rather big... does this increase the size of `InterpError`?

Reply:
Yeah, it increased it from 32 bytes to 64 bytes. I'll put it behind a Box to bring it back down.

Reply:
Could you please add a `static_assert_size!` to the error type?